testharness/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     testImplementation 'org.mockito:mockito-inline:5.2.0'
 
     jacocoTestReport {
+        dependsOn(':service:compileTestJava')

```

Comment:
Would someone be able to try building LZS with and without this change (`main` and this branch should do it) to see if it was an issue with my machine or not?